commons/connector/upload/physilog/physilog_binary_data.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.connector.upload.physilog",
+  "type": "record",
+  "name": "PhysilogBinaryData",
```

Comment:
Don't know much about this but would `PhysilogBinaryDataReference` be better suited name? And same for the topic in specs?